components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -533,8 +544,16 @@ export default class Profile extends mixins(
     })
   }
 
-  get isBsx(): boolean {
-    return this.urlPrefix === 'bsx'
+  get isAllowSetIdentity(): boolean {
```

Comment:
Move to utils pls

Reply:
this is a `getter` only ProfileDetail use.